### PR TITLE
Fix for SAK-29192. Chat messages not refresh on submission

### PR DESCRIPTION
**Pull request by willkara**
Redid the PR correctly this time.

Chat messages now correctly refresh when submitted.

### CODE REVIEW

File: reference/library/src/webapp/js/headscripts.js [hunk 1]
```diff
@@ -496,7 +496,7 @@ function showNotif(item, button,formName)
 }
 
 // stuff to do auto-update using the XMLHttpRequest object
-var updateReq;
+var updateReq = null;
```
Comment (buckett) [inline reference/library/src/webapp/js/headscripts.js]: Isn't this the only bit of the change that is actually needed. I can't understand why any other changes are needed.

**Comment by willkara** [comment]
Hmmmm, you might be right.  I'll take a look at it when I get into work today.

**Comment by willkara** [comment]
@buckett , turns out you're right. I'll close the PR and submit another. Since it is in head scripts, do you foresee any problems with this change?

**Comment by buckett** [comment]
@willkara I don't foresee any problems with that.

**Pull request closed without merging**